cardinal/server/server.go

```diff
@@ -39,8 +39,8 @@ var (
 // NewHandler instantiates handler function for creating a swagger server that validates itself based on a swagger spec.
 // transactions and queries registered with the given world are automatically created. The server runs on a default port
 // of 4040, but can be changed via options or by setting an environment variable with key CARDINAL_PORT.
-func NewHandler(w *ecs.World, opts ...Option) (*Handler, error) {
-	h, err := newSwaggerHandlerEmbed(w, opts...)
+func NewHandler(w *ecs.World, builder *middleware.Builder, opts ...Option) (*Handler, error) {
```

Comment:
Since the builder parameter is optional, why not make it an Option?

Reply:
because of the way it's passed in. Options in server operate on the handler overall, but the builder parameter needs to be passsed into another method that's activated on a route handler.

Reply:
👍